sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,14 +115,14 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/callumfare/unified-runtime.git")
   # commit b766009add8dc41ad03e2a63be9f6ed40eea4c55 (HEAD, origin/main, origin/HEAD)
   # Merge: 608e9184 c3c57284
   # Author: Omar Ahmed <omar.ahmed@codeplay.com>
   # Date:   Tue Aug 27 16:30:04 2024 +0100
   #   Merge pull request #2011 from nrspruit/fix_opencl_usm_align_check
   #   [OpenCL] Fix USM alignment error check to occur always and return nulllptr
-  set(UNIFIED_RUNTIME_TAG b766009add8dc41ad03e2a63be9f6ed40eea4c55)
+  set(UNIFIED_RUNTIME_TAG callum/ur_api_def)
```

Comment:
Do you have a link to this UR PR?

Reply:
https://github.com/oneapi-src/unified-runtime/pull/2045

Reply:
thanking you